docker-compose.yml

```diff
@@ -1,78 +1,94 @@
 # limitations under the License.
+################################################################################
 
 version: "2.1"
+
 services:
+
+  ###############################################################
+  #    StateFun runtime
+  ###############################################################
+
+  statefun-manager:
+    image: apache/flink-statefun:3.0.0
+    expose:
+      - "6123"
+    ports:
+      - "8081:8081"
+    environment:
+      ROLE: master
+      MASTER_HOST: statefun-manager
+    volumes:
+      - ./module.yaml:/opt/statefun/modules/workshop/module.yaml
+
+  statefun-worker:
+    image: apache/flink-statefun:3.0.0
+    expose:
+      - "6121"
+      - "6122"
+    environment:
+      ROLE: worker
+      MASTER_HOST: statefun-manager
+    volumes:
+      - ./module.yaml:/opt/statefun/modules/workshop/module.yaml
+
+
+  ###############################################################
+  #    Kafka for ingress and egress
+  ###############################################################
+
   zookeeper:
     image: wurstmeister/zookeeper
     ports:
       - "2181:2181"
+
   kafka-broker:
     image: wurstmeister/kafka:2.12-2.0.1
     ports:
       - "9092:9092"
     environment:
       KAFKA_ADVERTISED_HOST_NAME: "kafka-broker"
       HOSTNAME_COMMAND: "route -n | awk '/UG[ \t]/{print $$2}'"
-      KAFKA_CREATE_TOPICS: "transactions:1:1,alerts:1:1,confirmed:1:1,thresholds"
+      KAFKA_CREATE_TOPICS: "transactions:1:1,alerts:1:1,confirmed:1:1,thresholds:1:1"
       KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
     depends_on:
       - zookeeper
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
-  simulator:
-    build:
-      context: simulator 
-      dockerfile: Dockerfile
-    depends_on:
-      - kafka-broker
-  master:
-    build:
-      context: statefun-runtime
-    # uncomment and adjust savepoint path to start from a savepoint
-    # command: -s /savepoint-dir/savepoint-3df90d-aa82d691740f
-    expose:
-      - "6123"
-    ports:
-      - "8001:8001"
+
+  ###############################################################
+  # Forward a port 8000 from the host's machine
+  ###############################################################
+
+  host-machine:
+    image: qoomon/docker-host@sha256:e0f021dd77c7c26d37b825ab2cbf73cd0a77ca993417da80a14192cb041937b0
+    cap_add: [ 'NET_ADMIN', 'NET_RAW' ]
+    mem_limit: 8M
+    restart: on-failure
     environment:
-      - ROLE=master
-      - MASTER_HOST=master
-    volumes:
-      - ./checkpoint-dir:/checkpoint-dir
-      - ./savepoint-dir:/savepoint-dir
-  worker:
-    build:
-      context: statefun-runtime
-    expose:
-      - "6121"
-      - "6122"
+      PORTS: 8000
+
+  ###############################################################
+  #    Simple Kafka JSON producer to simulate ingress events
+  ###############################################################
+
+  simulator:
+    image: statefun-workshop-simulator:1.0.0
```

Comment:
Once #11 is published I will update this so the image does not need to be built locally